### DISCUSSION TITLE
How does translations work and how can i translate globaleaks in a new language?

### DISCUSSION THREAD

**Discussion by jurdajrd** [globaleaks/globaleaks-whistleblowing-software#4550] [Q&A] [answered]
**Describe the bug**
I am using whistleblowing in Czech. I want to translate some sentences which are important from English to Czech and I press save button a lot of times, but it is not working and it is still in english.

**Expected behavior**
I guess senteces should be in Czech when I translate them.

**Screenshots**
![preklad](https://user-images.githubusercontent.com/54391873/159254116-4283c10f-4dcc-42f6-ad66-c4b905eb33d6.png)

![neprelozeno](https://user-images.githubusercontent.com/54391873/159254134-9ca8896a-d38a-46fe-8a78-b0d0f4c596f4.png)

**Comment by evilaliv3** [comment, marked as answer]
Thank you for your reporting @jurdajrd 

Actually the feature for custom translation is not available for the default questionnaire and is for this reason that it do not work. It works only for some of the texts of the application.

The completion of the Czech translation should be done on https://www.transifex.com/otf/globaleaks/dashboard/

If you would like to contribute to it please register an account on Transifex and reach out to me communicating me your usernam at my email: giovanni.pellerano@globaleaks.org so that i could grant you privileges for this.

**Comment by jurdajrd** [comment]
I send you an email with my username. When I complete translation for Czech, when it will be applied in our verison? After update? 

Thanks.

**Comment by evilaliv3** [comment]
The will be updated automatically after updating to the next software release.

We typically issue new software update as soon that it is valuable to, that means users could benefit of a new translation or of the completion of a new valuable feature.
